PA1fibo

```diff
@@ -0,0 +1,96 @@
+# COMP3410 Program Template
+# Author: Will Robb
+# Assignment: PA[1]
+# Date: 2/11/15
+
+# Turn in one .asm file per assignment component
+# Remember to submit it as a pull request to the GitHub repo for the assignment
+
+# fibonacci.asm
+# Input: None
+# Output: Print to terminal N Fibonacci numbers
+
+##########################################################
+# .data segment. Constant and variable definitions go here.
+##########################################################
+
+	.data
+
+fibs:	.word   0 : 9         # create an array variable named "fibs" of 9 word-length elements (4 bytes each)
+prompt: .asciiz "Enter the size of the array to print: "
+
+###############################################
+# .text segment. Assembly instructions go here.
+###############################################
+	.text
+	      la   $s0, fibs        # load address of array into $s0
+	      
+input:	li $v0, 4		    # prepare to print string to ask for input
+	la $a0, prompt		    # load prompt in memory
+	syscall			    # print
+	
+	li $v0, 5		    # prepare the system to read in variable
+	syscall
+	
+	add $s5, $s5, $v0	    # move the variable into a register
+	bgt $s5, 9, input	    # branch to input if the input is greater than 9
```

Comment:
-5 Your fibonacci program gets stuck in an infinite loop if user enters input > 9!  Make sure to test thoroughly before submitting.